src/PdoStreamIterator.php

```diff
@@ -228,6 +228,11 @@ public function rewind(): void
         }
     }
 
+    public function count(): int
```

Comment:
There is a problem with this approach. It will return count only for current batch size, not count for the iterator. This can lead to users using this method and receiving unexpected results.

Can we make it private and call it via reflection api in the tests?

/cc @bweston92 @basz